.github/workflows/rebase.yaml

```
@@ -8,6 +8,7 @@ permissions:
   id-token: write
   contents: write
   pull-requests: write
+  workflows: write
```

actionlint (1.7.12)

[error] unknown permission scope "workflows". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)